config/routes.rb

```diff
@@ -185,6 +185,10 @@
 
       scope module: "providers" do
         resources :allocations, only: %i{index}, on: :member, param: :training_provider_code do
+          collection do
+            get :new_request, path: "request"
+          end
```

Comment:
This new route gives provides us with:

`GET /organisations/[accredited_body_code]/2020/new-request`
and
`GET /organisations/[accredited_body_code]/2020/places`

We currently have the following routes for **re-requesting / renewing** an allocation
`GET /organisations/[accredited_body_code]/2020/allocations/[training_provider_code]/new`

and 

`POST /organisations/[accredited_body_code]/2020/allocations/[training_provider_code]` 

I think regardless of the route pattern we go for we should aim to have consistency across both the request and re-request / renewal process.

The new collection route does not have allocation in the path ...

Would the following be a viable alternative?

`GET /organisations/[accredited_body_code]/2020/allocations/new-request`

`GET /organisations/[accredited_body_code]/2020/allocations/places`

`GET /organisations/[accredited_body_code]/2020/allocations/[training_provider_code]/renew-request`
I've opted for 'renew-request` over `re-request` as we discussed (Aga and Graeme) that this may be clearer. Rather than going to `new` controller action, it could go to a `renew_request` action, again for consistency.

`POST /organisations/[accredited_body_code]/2020/allocations/[training_provider_code]`

Reply:
Happy to change it to `renew_request`. When I run it locally I have:
`GET /organisations/[accredited_body_code]/2020/allocations/request`

<img width="543" alt="Screenshot 2020-04-28 at 09 18 14" src="https://user-images.githubusercontent.com/38078064/80464212-5f178080-8931-11ea-8791-e0ebf94694de.png">
I think you may have the old WIP version running.